kornia/geometry/transform/imgwarp.py

```diff
@@ -56,7 +57,9 @@ def warp_perspective(
         M: transformation matrix with shape :math:`(B, 3, 3)`.
         dsize: size of the output image (height, width).
         mode: interpolation mode to calculate output values ``'bilinear'`` | ``'nearest'``.
-        padding_mode: padding mode for outside grid values ``'zeros'`` | ``'border'`` | ``'reflection'``.
+        padding_mode (str): padding mode for outside grid values
+          ``'zeros'`` | ``'border'`` | ``'reflection'`` | ``'fill'``.
+        padding_tensor: tensor of shape :math:`(3)` that fills the padding area. Only supported for RGB.
```

Comment:
probably fill_value or something instead? padding_tensor reads confusing to me.

Reply:
Updated to `fill_value`